protocol/migrations/sqlite/1676315400_add_deleted_column_to_activity_center_notifications_table.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE activity_center_notifications ADD COLUMN deleted BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Does `DEFAULT` constraint update existing columns? E.g do we need migration setting `deleted=false` for existing notification?

Reply:
In this case no @MishkaRogachev. If it was an existing column, then yes, using the `DEFAULT` constraint would not suffice, and a migration would be required.